# Selection of Workflows for Processing Biological / Bioinformatics Data

<!-- GETTING STARTED -->

### Prerequisites

* Python > 3.0


<!-- USAGE EXAMPLES -->
## Contents and Usage 

* `blast-fasta.py` - Biopython program to iteratively run NCBI BLAST query on a set of input protein sequences
* `pdb_chain_select.py` - Biopython program to generate a new PDB file containing only the 
specified chain or chains of an existing PDB entry
* `sequences_csv_to_fasta.py` - Program to read in sequence data in NGS format and generate FASTA-formatted files for the corresponding sequences

<!-- ACKNOWLEDGEMENTS -->
## Acknowledgements
* [Biopython BLAST tutorial](https://biopython-tutorial.readthedocs.io/en/latest/notebooks/07%20-%20Blast.html)
* [Img Shields](https://shields.io)
* [Choose an Open Source License](https://choosealicense.com)
* 
